sw/device/lib/crypto/include/api.h

```diff
@@ -1225,7 +1277,7 @@ crypto_status_t otcrypto_x25519(const crypto_blinded_key_t *private_key,
  * started, or`kCryptoStatusInternalError` if the operation cannot be
  * started.
  *
- * @param required_key_len Requested key length in bits
+ * @param required_key_len Requested key length
```

Comment:
nit: (similar to `otcrypto_drbg_generate()`'s `output_len`'s doc comment)
```suggestion
 * @param required_key_len Requested key length, in bytes
```

Reply:
It's a bit different from `output_len` in that the requested length here has the type `rsa_key_size_t`, not `size_t`; we're not expecting the actual length, just a member of that specialized enum, so this phrasing makes more sense to me.